tests/test_freshdesk_pagination.py

```diff
@@ -1,58 +1,90 @@
-from tap_tester import menagerie, connections, runner
-import re
+from math import ceil
+from tap_tester import connections, runner, LOGGER
 
 from base import FreshdeskBaseTest
 
 class PaginationTest(FreshdeskBaseTest):
 
     def name(self):
-        return "tap_freshdesk_pagination_test"
+        return "tap_tester_freshdesk_pagination_test"
 
     def test_name(self):
-        print("Pagination Test for tap-freshdesk")
+        LOGGER.info("Pagination Test for tap-freshdesk")
 
     def test_run(self):
-
-        # instantiate connection
+        """
+        • Verify that for each stream you can get multiple pages of data.  
+        This requires we ensure more than 1 page of data exists at all times for any given stream.
+        • Verify by pks that the data replicated matches the data we expect.
+        """
+        
+        # For roles stream data present in test account is limited. So, adding configurable page_size "2" 
+        streams_to_test = {"roles"}
+        self.run_test(streams_to_test, 2)
+
+        # To collect "time_entries", "satisfaction_ratings" pro account is needed. Skipping them for now.
+        streams_to_test = self.expected_streams() - {"time_entries", "satisfaction_ratings"} - {"roles"}
```

Comment:
Mention the role in the comment and can we create only one list?

Reply:
Updated